server/src/test/java/com/google/finapp/FinAppIT.java

```diff
@@ -427,4 +429,49 @@ public void createTransactionForAccount_tooLargeAmount_throwsException() throws
         IllegalArgumentException.class,
         () -> spannerDao.createTransactionForAccount(accountId, amount, isCredit));
   }
+
+  @Test
+  public void getRecentTransactionsForAccount_valid() throws Exception {
```

Comment:
I'd change the second part of the function name to be a bit more specific. Maybe something like `returnsSingleTransaction`, and a future one can be `returnsMultipleTransactions`.